addons/60fps/addon.json

```
@@ -1,10 +1,15 @@
 {
-  "name": "60FPS project player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Custom project framerate",
+  "description": "Alt+Click the green flag to toggle between Scratch's default 30FPS and a custom framerate.",
   "info": [
+    {
+      "type": "info",
+      "text": "The framerate (FPS) determines how many blocks are run per second. For example, running at 60FPS doubles the execution speed.",
+      "id": "fpsExplanation"
+    },
     {
       "type": "notice",
-      "text": "Most projects will not behave properly when running at 60FPS, because increasing the frame rate also increases the speed at which scripts run. This should only be used on projects that support 60FPS.",
+      "text": "Most projects will not behave properly when running at different FPS, because increasing the frame rate also increases the speed at which scripts run. This should only be used on projects that support 60FPS.",
```

I have a rewrite for this:
```suggestion
      "text": "Changing the framerate of Scratch projects also affects the rate at which certain code (such as loops) runs, so most projects won't behave properly when run at a different framerate. This addon is meant to be used on projects that are designed to support a higher framerate.",
```